dev/openmage/apache/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ghcr.io/colinmollenhour/docker-openmage-dev:8.2-apache
+
+ENV APACHE_DOCUMENT_ROOT /var/www/html/pub/default
+
+RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/sites-available/*.conf
+RUN sed -ri -e 's!/var/www/!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.conf /etc/apache2/conf-available/*.conf
```

Comment:
are the changes to dev/openmage intended for this PR?

Reply:
Yes, this updates dev/openmage to use the new root directory by default.